### DISCUSSION TITLE
[Error] Symbol not found: _pthread_jit_write_protect_supported_np

### DISCUSSION THREAD

**Discussion by rodshtein** [JonasKruckenberg/imagetools#478] [Q&A]
If some one got the same error, do:

npm uninstall sharp
npm install sharp@0.31.1

Related: https://github.com/lovell/sharp/issues/3438

**Comment by jnaklaas** [comment]
Same here, but everytime I try to install sharp@0.31.1, npm creates a `node_modules/imagetools-core/node_modules/sharp` directory, containing sharp@0.31.3. So the error persists.

Unless I manually remove the sharp package from the imagetools-core package.
